v03_pipeline/lib/annotations/vep.py

```diff
@@ -78,6 +78,7 @@ def vep_110_transcript_consequences_select(
             ),
             c.intron.split('/').map(hl.parse_int32),
         ),
+        refseq_transcript_id=gencode_ensembl_to_refseq_id_mapping.get(c.transcript_id),
```

Comment:
There's a space saving move here... "set `refseq_transcript_id` to empty if `mane_select` is defined and equal to the result".  It's likely more confusing to the reader code though.

Reply:
decided against this.